pre-commit

```diff
@@ -10,6 +10,8 @@ if [ -L "$0" ]; then
 	else
 		DEV_LIB_PATH=$( dirname $( readlink "$0" ) )
 	fi
+elif [ -f "$0" ]; then
```

Comment:
This file `pre-commit` isn't a symlink when running in Vagrant so we check for a regular file too.